plugins/modules/aci_interface_selector_to_switch_policy_spine_profile.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Bruno Calogero <brunocalogero@hotmail.com>
+# Adapted from aci_interface_selector_to_switch_policy_spine_profile
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "certified",
+}
+
+DOCUMENTATION = r"""
+---
+module: aci_interface_selector_to_switch_policy_spine_profile
+short_description: Bind interface selector profiles to switch policy spine profiles (infra:RsSpAccPortP)
+description:
+- Bind interface selector profiles to switch policy spine profiles on Cisco ACI fabrics.
+options:
+  spine_profile:
+    description:
+    - Name of the Spine Profile to which we add a Selector.
+    type: str
+    aliases: [ spine_profile_name ]
+  interface_selector:
+    description:
+    - Name of Interface Profile Selector to be added and associated with the Spine Profile.
+    type: str
+    aliases: [ name, interface_selector_name, interface_profile_name ]
```

Comment:
I think we should remove name from the aliases since both paramaters are a name. What's your opinion on this?